developer/src/tike/xml/layoutbuilder/builder.js

```diff
@@ -744,15 +745,27 @@ $(function() {
     builder.keyCapChange(val);
   }, {saveOnce: true});
 
-  builder.keyHintChange = function(val) {
-    const key = builder.selectedKey();
-    key.data('hint', val);
-    $('.hint', key).text(builder.inferKeyHintText(val, $(key).data('longpress'), $(key).data('flick'), $(key).data('multitap')));
+  builder.updateHint = function(key) {
+    const val = key.data('hint');
+    const hintElement = $('.hint', key);
+    hintElement.text(builder.inferKeyHintText(val, $(key).data('longpress'), $(key).data('flick'), $(key).data('multitap')));
```

Comment:
Does `$(key).data('longpress')` fallback to the current default `.` if nothing is specified?

Reply:
`inferKeyHintText` does that.

https://github.com/keymanapp/keyman/blob/e6db899453607b6c4de1c72656dfae6a41c15962/developer/src/tike/xml/layoutbuilder/builder.js#L299-L306